docs/src/guides/languages/rust.md

```diff
@@ -81,15 +81,15 @@ By default `toolchain` setup to `rust-toolchain.toml`.
 
 ```Earthfile
 # Test rust build container - Use best architecture host tools.
-check-hosted:
+hosted-check:
```

Comment:
Why did these all get reversed?

Reply:
we want this check to be done locally. Already discuss with @stevenj